samples/app-anonymous-users/csharp/ClientApp/src/components/shareview.jsx

```diff
@@ -19,11 +20,30 @@ const ShareView = () => {
     const [uShowCount, uShowSetCount] = useState(0);
 
     // Declare new state variables that are required to disable the submit vote button.
-    const [disabled, setDisabled] = useState(false)
+    const [disabled, setDisabled] = useState(false);
+
+    // Declare new state variables that are required to assgin facebook user name
+    const [userName, setUserName] = useState("");
+
+    // Declare new state variables that are required to disable the Facebook button after login
+    const [disabledfacebookBtn, setdisabledfacebookBtn] = useState(true);
+
+    // Declare new state variables that are required for login success load the submit vote button
+    const [enableVoteDiv, setEnableVoteDiv] = useState(false);
+
+    // Declare new state variables that are required for a verified anonymous user or a normal user
+    const [IsCheckLoggeduser, IsSetCheckLoggeduser] = useState(true);
+
+    // Declare new state variables that are required for a verified anonymous user or a normal user
+    const [IsVisibleConsent, IsSetVisibleConsent] = useState(false);
+
+    // Declare new state variables that are required disable the authentication button after login
+    const [ssoAuthenticationBtn, ssoSetAuthenticationBtn] = useState(true);
```

Comment:
nit: should this be setSsoAuthenticationBtn instead of ssoSetAuthenticationBtn

Reply:
Updated your feedback.